ios/ReactNativeCameraKit/CKCameraViewComponentView.mm

```diff
@@ -165,6 +165,11 @@ - (void)updateProps:(const Props::Shared &)props oldProps:(const Props::Shared &
         _view.flashMode = [flashMode isEqualToString:@"auto"] ? CKFlashModeAuto :  [flashMode isEqualToString:@"on"] ? CKFlashModeOn : CKFlashModeOff;
         [changedProps addObject:@"flashMode"];
     }
+    id maxPhotoQualityPrioritization = CKConvertFollyDynamicToId(newProps.maxPhotoQualityPrioritization);
+    if (maxPhotoQualityPrioritization != nil && [maxPhotoQualityPrioritization isKindOfClass:NSString.class]) {
+        _view.maxPhotoQualityPrioritization = [flashMode isEqualToString:@"balanced"] ? CKMaxPhotoQualityPrioritizationBalanced :  [flashMode isEqualToString:@"quality"] ? CKMaxPhotoQualityPrioritizationQuality : CKMaxPhotoQualityPrioritizationSpeed;
+        [changedProps addObject:@"flashMode"];
+    }
```

Comment:
This looks like a mistake? changedProps is adding flashMode instead of maxPhotoQualityPrioritization. The code above also compares the flashmode to balanced. It looks like a simple copy-paste issue but this needs to be tested.

Can you please review?

Reply:
Looking at https://github.com/teslamotors/react-native-camera-kit/blob/c718fb5dc7e199fd307c39f43b3e02973ba3e701/ios/ReactNativeCameraKit/CameraView.swift#L200C35-L200C64 it seems that you are right and it was copy/paste mistake. Fixed, thanks!